addons/binding/org.openhab.binding.powermax/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Powermax Binding
+Bundle-SymbolicName: org.openhab.binding.powermax;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
+ gnu.io,
```

Comment:
Instead of using gnu.io, could you switch to the ESH serial API instead?

Reply:
Don't want to change that now with the risk to break all the serial stuff.
Maybe in the future when we will have robust bindings using this new API.